aws/cta
# Check existence of files and directories before attempting deletion

## Related issue

Closes: #ISSUE_NUMBER_HERE


## Description
* During Web Forms conversion, check that directories and files exist before attempting deletion. This will prevent excessive error logging.
* Upgrade nuget packages `NJsonSchema` and `HTMLAgilityPack` to their latest versions
* Resolve race condition where test project files used in the CTA conversion tests were being deleted by the Web Forms conversion tests, causing unit test failures in the CI/CD pipeline

## Supplemental testing
* Manually tested
* Unable to add unit test for exception throwing because the exception is caught within the function

---
*By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.*
